src/server/scripts/OutdoorPvP/OutdoorPvPNA.cpp

```diff
@@ -165,7 +218,7 @@ void OPvPCapturePointNA::FactionTakeOver(TeamId teamId)
         _pvp->SendUpdateWorldState(NA_UI_GUARDS_LEFT, m_GuardsAlive);
         sWorld->SendZoneText(NA_HALAA_GRAVEYARD_ZONE, sObjectMgr->GetAcoreStringForDBCLocale(LANG_OPVP_NA_CAPTURE_A));
     }
-    else
+    else if(teamId == TEAM_HORDE)
```

Comment:
Do we need else if? I dont think it can be anything other than horde in this case